cms/djangoapps/contentstore/views/course.py

```diff
@@ -792,6 +793,7 @@ def settings_handler(request, course_key_string):
                 'test_email_url': reverse_course_url('send_test_enrollment_email', course_key),
                 'default_pre_template': default_enroll_email_template_pre,
                 'default_post_template': default_enroll_email_template_post,
+                'keywords_supported': [{'name':keyword, 'desc': value.desc} for keyword, value in KEYWORD_FUNCTION_MAP.iteritems()],
```

Comment:
Actually, since we're using this same comprehension multiple times, would it make sense to wrap it in a helper function, `get_keywords_supported`?